lib/pushmi_pullyu.rb

```diff
@@ -43,11 +43,12 @@ module PushmiPullyu
       base_path: '/dev'
     },
     swift: {
-      auth_version: 'v1.0',
       tenant: 'tester',
       username: 'test:tester',
       password: 'testing',
-      endpoint: 'http://localhost:8080',
+      auth_url: 'http://localhost:8080/auth/v1.0',
```

Comment:
Since we are not using v1.0, I think the example used in these configurations should set up to as close to our true configuration as possible to avoid confusion?

Reply:
Never mind. This is used to facilitate testing which builds in v1.0? Is it possible to upgrade our swift testing env to use the latest version, so it is actually useful?

Reply:
our docker image uses v1.0 authentication (config files authentication vs keystone). This is why it is v1.0 keystone would require a bit to work to be installed inside docker image.